WordPress/Classes/Models/BasePost.m

```diff
@@ -115,6 +123,18 @@ - (NSString *)statusTitle
     return [BasePost titleForStatus:self.status];
 }
 
+// If the post has a future publish date. This is different from "isScheduled" in that
```

Comment:
Nitpicky but... Xcode recognizes documentation comments with either `/**` or `///` (you can check this by using a different comment color for documentation comments in Xcode).

I'd suggest for consistency to use one of those instead for these comments since they're describing the method.